### DISCUSSION TITLE
Does Homebrew support iOS SDK?

### DISCUSSION THREAD

**Discussion by paperchalice** [Homebrew/discussions#1091] [Everyday usage]
If a formula depends on `xcode`, it can use iOS (or tvOS watchOS) SDK theoretically, but the shims seems is designed only for the macOS. Is there any extra steps to use iOS SDK?

**Comment by Bo98** [comment]
I haven't tried but I don't imagine it being possible. It's a situation I don't really understand the use case for - why do you want to build and ship iOS apps/binaries through a macOS package manager?

SDK is one thing, but there's the whole concept of cross-compiling to deal with first. If you are on an Intel Mac then `brew` will build x86_64 binaries - but you want arm64 for iOS. Also codesigning etc may require handling.
